src/main/java/ch/njol/skript/expressions/ExprItemWithTooltip.java

```diff
@@ -0,0 +1,76 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemFlag;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Item with Tooltip")
+@Description({
+	"Get an item with or without entire/additional tooltip.",
+	"If changing the 'entire' tooltip of an item, nothing will show up when a player hovers over it.",
+	"If changing the 'additional' tooltip, only specific parts (which change per item) will be hidden."
+})
+@Examples({
+	"set {_item with additional tooltip} to diamond with additional tooltip",
+	"set {_item without entire tooltip} to diamond without entire tooltip"
+})
+@RequiredPlugins("Spigot 1.20.5+")
```

Comment:
```suggestion
@RequiredPlugins("Minecraft 1.20.5+")
```
the method/expression isnt exclusive to spigot